src/pysdmx/io/xml/__structure_aux_reader.py

```diff
@@ -551,8 +551,8 @@ def __format_con_id(self, concept_ref: Dict[str, Any]) -> Dict[str, Any]:
         return rep
 
     @staticmethod
-    def __format_relationship(json_rel: Dict[str, Any]) -> Optional[str]:
-        att_level = None
+    def __format_relationship(json_rel: Dict[str, Any]) -> str:
```

Comment:
Also, I'm not sure what is the intent to call an attribute json_rel in a module dealing with XML parsing. What does this attribute hold?